pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+python_files = tests/tests.py tests/test_expand.py tests/test_io.py tests/test_schema.py tests/test_linting.py tests/test_executor_test_suite.py
```

Comment:
The test suite contains plenty of files which doesn't follow the standard pytest conventions. This tripped the vs-code test plugin which tried to inspect non-test files which it thought caused it to crash.